pymongo/helpers.py

```diff
@@ -210,7 +238,7 @@ def _raise_write_concern_error(error: Any) -> NoReturn:
     raise WriteConcernError(error.get("errmsg"), error.get("code"), error)
 
 
-def _get_wce_doc(result):
+def _get_wce_doc(result: Dict[str, Any]) -> Optional[Dict[str, Any]]:
```

Comment:
The argument type should be `Mapping[str, Any]` to be consistent with https://docs.python.org/3/library/typing.html#typing.Dict.